### PR TITLE
Maturity levels

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by psmulovics** [comment]
@jstclair2019 you really have to clear that CLA :D

**Comment by jstclair2019** [comment]
> @jstclair2019 you really have to clear that CLA :D
@psmulovics argh, yeah I know! 😄

**Comment by robmoffat** [comment]
Hi All,

Check out the maturity levels I've added, starting here:

https://deploy-preview-103--open-source-readiness.netlify.app/docs/bok/OSMM/Introduction

I'm aiming to reconcile both TODO OSPO maturity and OSPO Alliance levels.  Also, I've added indexing so you should be able to see which activities correspond to which levels (this will be important as we complete the work on the developer maturity model in the SIG)

thanks!

File: docs/bok/Activities/Security-Testing.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Security Testing
+tags: 
+  - OSPO (Role)
+  - Developer (Role)
+  - Security Expert (Role)
+  - Placeholder
```
Comment (psmulovics) [inline docs/bok/Activities/Security-Testing.md:7]: Is this a placeholder entry? Seems pretty well written for that.

Comment (jstclair2019) [reply docs/bok/Activities/Security-Testing.md:7]: @psmulovics I think so. We want to allow for additional roles as we define them (security expert could delineate into multiple roles in CISO, vulnerability management, etc)

Comment (psmulovics) [reply docs/bok/Activities/Security-Testing.md:7]: That is creating a tag called 'Placeholder' and adds this document to it :) So it's not a placeholder for the role.

Comment (robmoffat) [reply docs/bok/Activities/Security-Testing.md:7]: Hi yes - I've tagged all the documents "Placeholder" where we need to add some decent content.

For this one, I think there are several roles Jim came up with (CISO, TISO etc) that have might roll under the category of "Security Expert". What do you think?

**Comment by robmoffat** [comment]
@jstclair2019 if you go onto your LFX profile and associate with the Linux Foundation you might be able to get the green light for the EasyCLA process - should be something you have to accept.

**Comment by jstclair2019** [comment]
> @jstclair2019 if you go onto your LFX profile and associate with the Linux Foundation you might be able to get the green light for the EasyCLA process - should be something you have to accept.

LFX walked me through the same contributor process that you did, so it should be sitting in @maoo or someone's queue to approve..?

File: docs/bok/OSMM/Introduction.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ tags:
...
 
- ## Why Have an Open Source Maturity Model?
+## Why Have an Open Source Maturity Model?
 
 - The OSMM framework can help organizations to better understand the benefits and risks of using open source software and to establish policies and procedures to manage these effectively. 
 - The model can also be used as a tool for benchmarking an organization's open source maturity against that of other organizations in the same industry or sector. 
 - The OSMM can provide guidance to organizations on how to improve their open source practices and to align these with their overall business objectives. 
 - The ultimate goal of an OSMM is to enable organizations to maximize the benefits of open source software while managing the associated risks and costs effectively.
+- Since there are lots of [Activities](../Activities/Introduction) in this body of knowledge, the maturity levels provide some guidance about the _order_ to tackle activities.  Organisastions beginning their open source journey are advised to start with activities categorized as [Level 1](Level-1) and proceed from there.
```
Comment (psmulovics) [inline docs/bok/OSMM/Introduction.md:17]: typo: Organisastions

Comment (robmoffat) [reply docs/bok/OSMM/Introduction.md]: ta

**Review by psmulovics — APPROVED** [review]

**Pull request merged**